packages-exp/auth-exp/src/platform_cordova/popup_redirect/popup_redirect.ts

```diff
@@ -64,6 +109,72 @@ class CordovaPopupRedirectResolver implements PopupRedirectResolver {
   ): void {
     throw new Error('Method not implemented.');
   }
+
+  private attachCallbackListeners(auth: Auth, manager: AuthEventManager): void {
+    const noEvent: AuthEvent = {
```

Comment:
nit: You may want to make a helper function for this, since `_createError` does call `new Error()` and can be potentially expensive (capturing the call stack).